playkit/src/main/java/com/kaltura/playkit/plugins/ads/AdEvent.java

```diff
@@ -142,9 +144,9 @@ public AdRequestedEvent(String adTagUrl) {
 
         public final String clickThruUrl;
 
-        public AdClickedEvent(String clickThruUrl) {
+        public AdClickedEvent(@Nullable String clickThruUrl) {
             super(Type.CLICKED);
-            this.clickThruUrl = clickThruUrl;
+            this.clickThruUrl = (clickThruUrl != null) ? clickThruUrl : "";
```

Comment:
I think it's better to use null as optional, not empty string.

Reply:
This is used to display the value - if it is null or "" it is same no meaning except that the receiving side will not have for null